### ISSUE TITLE
PW Sprint 79 plan

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#12695]
## Q1 Priorities
_Items with [x] are carried over from previous sprint_

### 1. Post-MVP Homepage 
**General:**
- [ ] #12770 (3) - FE
- [ ] (depends on greenlight) #12160 (2) - FE
- [ ] #12790 (3) - Research
- [ ] #12814 (5) - Research

**"Create Account" block:**
  - [x] (stretch) #12532 (3) - FE (Daniel?)

### 2. Header/footer redesign
- Awaiting info / green light for kickoff

### 3. Income Limits web app
**Design / Research:**

All of the following are blocked pending stakeholder review/verification:
- [ ] (no longer blocked on SMEs; meeting Danielle on 3/1 for her timeline) #12341 (5) - Design
- [ ] (blocked on prototype finalization) #12256  (5, includes launching recruiting) - Research
- [ ] (blocked on prototype finalization) #12248 (3) - Research
- [ ] (blocked on prototype finalization; not ticketed) Midpoint review - Include LauraF - Research
- [ ] (blocked on prototype finalization) #12340 (5) - Research
- [ ] (blocked on prototype finalization) #12343 (5) - Research

**Engineering:**
- [ ] (Stretch, blocked on Plat implementation, may unblock in sprint 79) #12348 (5) - BE
- [ ] (stretch) #12402 (8) - Ruby

### 4. Benefit Content Model Hardening 
- [x] #12731 (5) - BE
- [ ] (blocked; DaveP not avail s79) #12736 (5) - Content
- [ ] (stretch) #12735 (needs refining) - Content 

### 5. PACT Act Wizard
- Pending movement from stakeholders; first ticket is ready for Cindy (research) 

### 6. Accordion header levels 
- [x] (carry-over) #12342 (5) - FE
- [x] (carry-over) #12345 (5) - FE
- [ ] (Laura F) #12352 (8) - 508
- [ ] (stretch; depends on testing) #12346 (2) - FE

### 7. Cerner API deprecation support
- [ ] #9307 (5) - [tasks remaining](https://github.com/department-of-veterans-affairs/vets-website/pull/23036#issuecomment-1448615887) - FE

___

##  Other sprint priorities 

- [x] (blocked on Platform) #12625 (5) - FE - hopefully will unblock in S79
- [ ] (stretch) #12339 (5) - FE
- [x] WIP #12568 (8) - BE

### Redirects & Content support
- [ ] #12794 (2) - FE/BE
- [ ] #12697 (2) - FE
- [ ] #12694 (5) - FE - 2nd week of sprint
- [ ] (stretch; added mid-s79) #12841

### Tech debt & Defects
- [X] (carryover from s78) #10307 (4 remain) - FE
- [ ] (stretch) #9469 (5) - BE
- [ ] #12800 - BE
- [ ] #12592 - BE

Epic of upcoming redirects not yet in our board: 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/53921

### BTSSS widget
- [ ] (blocked on content till end S79) #12591 (~8, tbd content) - FE

### Discharge wizard
_External dependencies, esp. SWContent – pending prioritization by DaveC for SWC_

### User research and UX pipeline development
- [ ] #12793 (3) - Research
- [ ] (stretch) #12791 (8) - Research
- [ ] (stretch) #12601 (5) - Research
- [ ] (stretch) #12719 (5) - Research

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [X] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by jilladams** [comment]
VA Logo alt text - we can include if we have capacity, not top priority: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12697

**Comment by wesrowe** [comment]
Adds from refinement:
- #12697 
-

**Comment by jilladams** [comment]
Refinement action items: 
* New ticket for work in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12423#issuecomment-1442298889
*

**Comment by jilladams** [comment]
Two Drupal bugs we could look at: 
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12792
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12800

**Comment by wesrowe** [comment]
DaveC said to prioritize spanish toggles, important for Veterans (and Congress) – demoted downtime alert discovery.
